Purchases/Public/RCPurchases.m

```diff
@@ -254,11 +253,9 @@ - (instancetype)initWithAPIKey:(NSString *)APIKey
                                                                      backend:backend
                                                         purchaserInfoManager:purchaserInfoManager];
     RCAttributionTypeFactory *attributionTypeFactory = [[RCAttributionTypeFactory alloc] init];
-    RCAttributionFetcher *attributionFetcher = [[RCAttributionFetcher alloc] initWithDeviceCache:deviceCache
-                                                                                 identityManager:identityManager
-                                                                                         backend:backend
-                                                                              attributionFactory:attributionTypeFactory
-                                                                                      systemInfo:systemInfo];
```

Comment:
Some of the stuff in `AttributionFetcher` was extracted into a new class, `AttributionPoster`, a while back, but its dependencies weren't updated. 
So I just removed the ones that aren't used by `AttributionFetcher` anymore.